### DISCUSSION TITLE
How do I set up multiple origins using CORS_ALLOW_ORIGIN?

### DISCUSSION THREAD

**Discussion by LukaszHem** [huggingface/text-embeddings-inference#275] [Q&A]
According to documentation at https://huggingface.co/docs/text-embeddings-inference/cli_arguments I can use CORS_ALLOW_ORIGIN to set up origin. How do I set up multiple origins?

**Comment by srjchauhan** [comment]
i am trying the cors-allow-origin to * but  i am still recieving cors error on browser

**Reply by LukaszHem** [reply]
I opened issue: https://github.com/huggingface/text-generation-inference/issues/1941

**Reply by LukaszHem** [reply]
Upvote please

**Reply by srjchauhan** [reply]
I did

**Reply by LukaszHem** [reply]
Thank you!
